Firebase/Auth/Source/FIRAuthAPNSTokenManager.m

```diff
@@ -137,7 +137,7 @@ + (BOOL)isProductionApp {
 
   NSError *error = nil;
 
-  Class envClass = NSClassFromString(@"FIRAppEnvironmentUtil");
+  Class envClass = NSClassFromString(@"GULAppEnvironmentUtil");
```

Comment:
Since Auth relies on `GoogleUtilities/GULAppEnvironmentUtil` already, can we remove the string based selectors here?